pkgs/os-specific/linux/powertop/default.nix

```diff
@@ -10,6 +10,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gettext libnl ncurses pciutils pkgconfig zlib ];
 
+  patchPhase = ''
+    substituteInPlace src/main.cpp --replace "/sbin/modprobe" "/run/booted-system/sw/bin/modprobe"
```

Comment:
I wonder, does this not tie the package to NixOS? There is certainly precedent in the repo for doing it like this, but what about using `module_init_tools` instead?